Dockerfile

```diff
@@ -1,78 +1,138 @@
-FROM python:3.7.1-stretch
+# Dockerfile contains multiple levels of images--
```

Comment:
Is the `base-image` cached then? I mean if I run this on my local machine and modify a layer after the base image has been created will it skip the creation of the base image?

Reply:
Yep! Unless you run with `--no-cache`